lint_typescript.json

```diff
@@ -1,153 +0,0 @@
-    "standard"
-  ],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
-    },
-    "ecmaVersion": 2018,
-    "sourceType": "module",
-    "project": "./tsconfig.eslint.json"
-  },
-  "rules": {
-    "arrow-parens": [
-      2,
-      "as-needed"
-    ],
-    "brace-style": [
-      2,
-      "stroustrup"
-    ],
-    "comma-dangle": [
-      2,
-      {
-        "arrays": "only-multiline",
-        "objects": "only-multiline",
-        "imports": "always-multiline",
-        "exports": "always-multiline"
-      }
-    ],
-    "indent": [
-      2,
-      2,
-      {
-        "flatTernaryExpressions": true
-      }
-    ],
-    "no-extra-parens": [
-      2,
-      "all",
-      {
-        "nestedBinaryExpressions": false
-      }
-    ],
-    "no-var": 2,
-    "object-curly-newline": 0,
-    "object-property-newline": 0,
-    "operator-linebreak": 0,
-    "padded-blocks": 0,
-    "quotes": [
-      2,
-      "single",
-      {
-        "allowTemplateLiterals": true
-      }
-    ],
-    "sort-imports": [
-      2,
-      {
-        "ignoreDeclarationSort": true
-      }
-    ],
-    "space-before-function-paren": 0,
-
-    // fp
-    "fp/no-class": 2,
-    "fp/no-delete": 2,
-    "fp/no-events": 2,
-    "fp/no-get-set": 2,
-    "fp/no-let": 2,
-    "fp/no-loops": 2,
-    "fp/no-mutating-assign": 2,
-    "fp/no-mutating-methods": 2,
-    "fp/no-this": 2,
-    "fp/no-valueof-field": 2,
-
-    // react
-    "react/jsx-curly-spacing": 2,
-    "react/jsx-equals-spacing": 2,
-    "react/jsx-tag-spacing": [
-      2,
-      {
-        "beforeSelfClosing": "allow"
-      }
-    ],
-    "react/no-children-prop": 0,
-    "react/no-unescaped-entities": 0,
-    "react/prop-types": 0,
-    "react/jsx-props-no-spreading": 0,
-    "react/button-has-type": 0,
-
-    // jsdoc
-    "jsdoc/check-alignment": 2,
-    "jsdoc/check-indentation": 2,
-    "jsdoc/check-syntax": 2,
-    "jsdoc/check-tag-names": 2,
-    "jsdoc/check-types": 2,
-    "jsdoc/implements-on-classes": 2,
-    "jsdoc/newline-after-description": 2,
-    "jsdoc/no-types": 2,
-    "jsdoc/no-undefined-types": 2,
-    "jsdoc/require-description-complete-sentence": [
-      2,
-      {
-        "abbreviations": ["e.g.","i.e."]
-      }
-    ],
-    "jsdoc/require-jsdoc": [
-      2,
-      {
-        "require": {
-          "ArrowFunctionExpression": true,
-          "ClassDeclaration": true,
-          "ClassExpression": true
-        }
-      }
-    ],
-
-    // typescript
-    "@typescript-eslint/semi": [2, "never"],
-    "@typescript-eslint/explicit-function-return-type": 0,
-    "@typescript-eslint/array-type": [2, {
-      "array-type": "array"
-    }],
-
-    // jsx
-    "jsx-quotes": [
-      "error",
-      "prefer-single"
-    ],
-
-    // import
-    "import/order": ["error", {
-      "groups": [
-        ["external", "builtin"],
-        ["internal", "index", "sibling", "parent"]
-      ],
-      "newlines-between": "ignore"
-    }]
```

Comment:
It appears that these rules have been removed and were not added in `.eslintrc.json`. Can you explain?

Reply:
I'll have to add `"@typescript-eslint/array-type"` and `"jsx-quotes"` rules back again. It was a manual error while I was trying to reduce the number of explicit rules. 

`semi` and `import/order ("sort-imports")` are a part of pre-existing eslint config already and they apply to ts files. And rule `"@typescript-eslint/explicit-function-return-type"` doesn't seem to throw an error anymore